documents/interaction-states.md

```diff
@@ -0,0 +1,51 @@
+# Interaction state machine
+
+During his work, a user can perform different actions, such as moving the mouse, leaving the seat, or typing on the keyboard. Due to the fact that those actions are immediate, it is possible to instantly transition from any state to any other.
+
+Each of the following states is defined in terms of those 3 parameters:
+
+- **Gaze**: workspace(💻), elsewhere (🌲), none (❌);
```

Comment:
**Gaze: none** as in the person is not visible on camera? I don't think there's such a thing as "no gaze", maybe we could rename this parameter as "physical presence" with options "looking at the screen", "looking elsewhere", "not present"?

Reply:
Yes, by **none** I meant that person is no visible on the camera. I agree that your wording is clearer.